modules/home-manager/zed-editor.nix

```
@@ -0,0 +1,21 @@
+{ config, lib, ... }:
+let
+  cfg = config.programs.zed-editor.catppuccin;
+  enable = cfg.enable && config.programs.zed-editor.enable;
+in
+{
+  options.programs.zed-editor.catppuccin = lib.ctp.mkCatppuccinOpt { name = "zed-editor"; } // {
+    noItalics = lib.mkEnableOption "no italics version of theme";
```

```suggestion
    italics = lib.mkEnableOption "the italicized version of the theme" // { default = true; };
```

Nitpick: I'm not a big fan of no/dont options. This might be a bit clearer